packages/core/src/services/scales-cartesian.ts

```diff
@@ -218,7 +231,7 @@ export class CartesianScales extends Service {
 	protected getScaleDomain(axisPosition: AxisPositions) {
 		const options = this.model.getOptions();
 		const axisOptions = Tools.getProperty(options, "axes", axisPosition);
-
+		const includeZero = axisOptions.includeZero === undefined || axisOptions.includeZero;
```

Comment:
why wouldn't we just do `axisOptions.includeZero` here? If it's undefined or null it'll be falsy, and if not it'll resolve

Reply:
The reason we do this check is that if the user does not set a value, `axisOptions.includeZero` will be undefined and it has to be treated as true by default.

We've added the logic to make possible to add default values into axes too. 
So now as default we set `axisOptions.includeZero` as true and we can just do `axisOptions.includeZero`.

Reply:
Can we please add a comment then saying what the `||` conditional is doing?

Reply:
👍